compiler/rustc_session/src/options.rs

```diff
@@ -1422,7 +1437,9 @@ options! {
         "inject the given attribute in the crate"),
     debug_info_for_profiling: bool = (false, parse_bool, [TRACKED],
         "emit discriminators and other data necessary for AutoFDO"),
-    debug_macros: bool = (false, parse_bool, [TRACKED],
+        debuginfo_compression: DebugInfoCompression = (DebugInfoCompression::None, parse_debuginfo_compression, [TRACKED],
+            "compress debug info sections (none, zlib, zstd, default: none)"),
+        debug_macros: bool = (false, parse_bool, [TRACKED],
```

Comment:
something went wrong with the indentation here

Reply:
I feel betrayed by rustfmt.

Fixed!